### PR TITLE
use six.moves for urllib

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

**Comment by emord** [comment]
is there a fixer for this now? For some reason I thought most of urrllib was a mnaul process

**Comment by nickpell** [comment]
@emord There is! 
https://github.com/python-modernize/python-modernize/blob/master/libmodernize/fixes/fix_urllib_six.py

However it was not included in the last pypi release of ```modernize```, so I had to clone the repo locally to access the fixer.

**Comment by emord** [comment]
@nickpell wut :cool: could you add the appropriate commit to dev-requirements? That seems like a reasonable place for that

**Comment by nickpell** [comment]
https://github.com/dimagi/commcare-hq/pull/18809

**Comment by nickpell** [comment]
✅

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import time
 import json
 import requests
-import urllib
+import six.moves.urllib.request, six.moves.urllib.parse, six.moves.urllib.error
```
[the remark this answers is not in the document]
Comment (emord) [reply corehq/apps/analytics/tasks.py]: We may want to ignore this in the stickler config? Seems like something we do a lot

**Review by emord — APPROVED** [review]

File: corehq/apps/analytics/tasks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (millerdev) [reply corehq/apps/analytics/tasks.py]: It's a [PEP-8 violation](https://www.python.org/dev/peps/pep-0008/#imports), fwiw. I didn't remember that we did this too much, although we frequently do `from x import y, z, ...` (not a PEP-8 violation).

[the remark this answers is not in the document]
Comment (emord) [reply corehq/apps/analytics/tasks.py]: Oh. yeah i was definitely thinking of the case `from x import y, z, ...`

**Pull request merged**